Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-grid-row/SubmissionGridRow.tsx

```diff
@@ -32,6 +32,11 @@ const SubmissionGridRow = ({ submission }: ISubmissionGridRowProps) => {
         isOfficial,
     } = submission;
 
+    const getProblemNumber = useMemo(
```

Comment:
`getProblemNumber` shows behavior, but it a state..

Reply:
Function removed.